ui/apps/platform/cypress/integration/vulnerabilities/nodeCves/cveDetailPage.test.ts

```diff
@@ -65,67 +94,239 @@ describe('Node CVEs - CVE Detail Page', () => {
+            visitNodeCvePage(mockCveName);
+            waitForRequests();
+
+            // check sorting of Node column
+            sortByTableHeader('Node');
+            waitAndYieldRequestBodyVariables().then(
+                expectRequestedSort({ field: 'Node', reversed: true })
+            );
+            sortByTableHeader('Node');
+            waitAndYieldRequestBodyVariables().then(
+                expectRequestedSort({ field: 'Node', reversed: false })
+            );
+
+            // check sorting of CVE Severity column
+            sortByTableHeader('CVE severity');
+            waitAndYieldRequestBodyVariables().then(
+                expectRequestedSort({ field: 'Severity', reversed: true })
+            );
+            sortByTableHeader('CVE severity');
+            waitAndYieldRequestBodyVariables().then(
+                expectRequestedSort({ field: 'Severity', reversed: false })
+            );
+
+            // check sorting of CVE status column
+            sortByTableHeader('CVE status');
+            waitAndYieldRequestBodyVariables().then(
+                expectRequestedSort({ field: 'Fixable', reversed: true })
+            );
+            sortByTableHeader('CVE status');
+            waitAndYieldRequestBodyVariables().then(
+                expectRequestedSort({ field: 'Fixable', reversed: false })
+            );
+
+            // check sorting of CVSS column
+            sortByTableHeader('CVSS score');
+            waitAndYieldRequestBodyVariables().then(
+                expectRequestedSort({ field: 'CVSS', reversed: true })
+            );
+            sortByTableHeader('CVSS score');
+            waitAndYieldRequestBodyVariables().then(
+                expectRequestedSort({ field: 'CVSS', reversed: false })
+            );
+
+            // check sorting of Cluster column
+            sortByTableHeader('Cluster');
+            waitAndYieldRequestBodyVariables().then(
+                expectRequestedSort({ field: 'Cluster', reversed: true })
+            );
+            sortByTableHeader('Cluster');
+            waitAndYieldRequestBodyVariables().then(
+                expectRequestedSort({ field: 'Cluster', reversed: false })
+            );
+
+            // check sorting of Operating System column
+            sortByTableHeader('Operating system');
+            waitAndYieldRequestBodyVariables().then(
+                expectRequestedSort({ field: 'Operating System', reversed: true })
+            );
+            sortByTableHeader('Operating system');
+            waitAndYieldRequestBodyVariables().then(
+                expectRequestedSort({ field: 'Operating System', reversed: false })
+            );
+
+            // check that the Affected components column is not sortable
+            queryTableHeader('Affected components');
+            queryTableSortHeader('Affected components').should('not.exist');
+        });
     });
 
     it('should filter the Node table', () => {
```

Comment:
Given that the data is mocked, there isn't much use for many of the tests described in the original comment block. Similarly, there are no per-row assertions done on the mocked data.